CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Contributor covenant [recently released](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) a v2.1 of their Code of Conduct. Example here you can reference for replacement: https://github.com/cisco-ospo/oss-template/blob/main/CODE_OF_CONDUCT.md

Reply:
I updated the CoC version at https://github.com/ossf/scorecard-monitor/pull/85/commits/f3706d5a2ba1dafe65263e9b64beaa91dee8c83a. Thanks for the suggestion!